openshift/e2e-common.sh

```diff
@@ -83,5 +106,15 @@ function install_knative_kafka() {
   sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-kafka-broker-receiver|${KNATIVE_EVENTING_KAFKA_BROKER_RECEIVER}|g"       ${DP_RELEASE_YAML}
 
   oc apply -f ${DP_RELEASE_YAML}
+  wait_until_pods_running $EVENTING_NAMESPACE || return 1
 }
 
+function run_e2e_tests(){
+
+  # go_test_e2e -timeout=30m ./test/e2e_new \
+  #   -imagetemplate "${TEST_IMAGE_TEMPLATE}" || fail_test "E2E (new) suite failed"
+
+
+  go_test_e2e -timeout=30m ./test/e2e/ \
+    -imagetemplate "${TEST_IMAGE_TEMPLATE}" || fail_test "E2E suite failed"
+}
```

Comment:
We need to set the Kafka Broker class by default:

```suggestion
function run_e2e_tests(){
  
  oc apply -f ./test/config/config-br-defaults.yaml
  
  # go_test_e2e -timeout=30m ./test/e2e_new \
  #   -imagetemplate "${TEST_IMAGE_TEMPLATE}" || fail_test "E2E (new) suite failed"


  go_test_e2e -timeout=30m ./test/e2e/ \
    -imagetemplate "${TEST_IMAGE_TEMPLATE}" || fail_test "E2E suite failed"
}
```